juju/juju
# LXD Provider - Apply Constraints to Containers

## Description of change

- Honours constraints for cores, memory and instance-type in the LXD provider.
- Removes handling for container constraint (it is always LXD).
- Moves a method.

## QA steps

- Bootstrap with _bootstrap-constraints_; check the created container for expected limits.
- Add a machine with _constraints_; check the created container for expected limits.
- Add a machine without specifying constraints; check that no limit config is present.

## Documentation changes

Should be covered under https://github.com/juju/juju/pull/8891.

## Bug reference

N/A
